### PR TITLE
Update Public Safety page

### PR DESCRIPTION
**Pull request by eneemann**
Includes updates for PSAP and Law Enforcement Boundaries made on 21 July 2021.

### CODE REVIEW

File: data/society/public-safety/index.html [hunk 1]
```diff
@@ -177,8 +178,8 @@ <h3 id="PSAPBoundaries">PSAP Boundaries</h3>
     <img class="inline-text-left outline" src="{% link images/PSAPBndy2.png %}" alt="PSAP Icon" loading="lazy" />
     <p>PSAP Boundaries represents Primary Safety Answering Point (PSAP) boundaries in Utah that will be used for
       call-routing in the Next-Generation 911 system. Countywide PSAP jurisdiction boundaries are based on county
-      boundaries, as of July 26, 2012. PSAP jurisdictions representing individual cities are based on municipal
-      boundaries, as of July 26, 2012. The data also includes jurisdictional boundary representations of the Navajo
+      boundaries, as of July 21, 2021. PSAP jurisdictions representing individual cities are based on municipal
```
Comment (steveoh) [inline data/society/public-safety/index.html:181]: should we remove this date or reference the updates somewhere so you don't need to baby sit it?

Comment (eneemann) [reply data/society/public-safety/index.html:181]: Good point, I'll reword it after lunch

**Review by steveoh — COMMENTED** [review]
I'm on crack

**Review by steveoh — APPROVED** [review]

**Comment by eneemann** [comment]
I wasn't sure what you meant, but I did add a couple more commits by editing in GitHub.

**Pull request merged**